common/rfb/SSecurityPlain.cxx

```diff
@@ -53,6 +55,13 @@ bool PasswordValidator::validUser(const char* username)
   for (size_t i = 0; i < users.size(); i++) {
     if (users[i] == "*")
       return true;
+#if !defined(WIN32) && !defined(__APPLE__)
```

Comment:
I don't suppose you checked if we can get Windows handling as well?

Reply:
I didn't build tigervnc for windows. Also as far as I can tell the only authentication methods supported on Windows are "None" and "Standard VNC". I didn't find any way to enable "Plain" authentication.